ffi/CMakeLists.txt

```diff
@@ -37,9 +37,11 @@ add_library(llvmlite SHARED assembly.cpp bitcode.cpp core.cpp initfini.cpp
             passmanagers.cpp targets.cpp dylib.cpp linker.cpp object_file.cpp)
 
 # Find the libraries that correspond to the LLVM components
-# that we wish to use
-# llvm_map_components_to_libnames(llvm_libs support core irreader)
-llvm_map_components_to_libnames(llvm_libs all)
+# that we wish to use.
+# TODO: the following line is broken with LLVM 10.0.0 due to a potential bug in
```

Comment:
Is this still a TODO?

Reply:
The TODO is to investigate this further and potentially submit a bug upstream to LLVM

Reply:
the link has been added.